proto/scene.proto

```diff
@@ -3,18 +3,18 @@ syntax = "proto3";
 package scene_format;
 
 message Scene {
-
-    message Vector3 {
-        double x = 1;
-        double y = 2;
-        double z = 3;
-    }
-
-    message Camera {
-        int32 id = 1;
-        Vector3 position = 2;
-    }
-
     int32 version = 1;
     repeated Camera cameras = 2;
 }
+
+message Vector3 {
+    double x = 1;
+    double y = 2;
+    double z = 3;
+}
+
+message Camera {
+    int32 id = 1;
+    Vector3 position = 2;
```

Comment:
What is 2? Shouldn't this be defined as (x, y, z)?

Reply:
This is protobuf field number. You can read more about it [here](https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers).